2022-03-15 15:19:29,166 INFO    MainThread:3820 [wandb_setup.py:_flush():75] Loading settings from C:\Users\Trevor\.config\wandb\settings
2022-03-15 15:19:29,166 INFO    MainThread:3820 [wandb_setup.py:_flush():75] Loading settings from F:\DirectedDistanceFunction\v5\wandb\settings
2022-03-15 15:19:29,166 INFO    MainThread:3820 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-15 15:19:29,166 INFO    MainThread:3820 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'v5\\train.py', 'program': 'train.py'}
2022-03-15 15:19:29,166 INFO    MainThread:3820 [wandb_init.py:_log_setup():386] Logging user logs to F:\DirectedDistanceFunction\v5\wandb\run-20220315_151929-3w2guk63\logs\debug.log
2022-03-15 15:19:29,166 INFO    MainThread:3820 [wandb_init.py:_log_setup():387] Logging internal logs to F:\DirectedDistanceFunction\v5\wandb\run-20220315_151929-3w2guk63\logs\debug-internal.log
2022-03-15 15:19:29,166 INFO    MainThread:3820 [wandb_init.py:init():420] calling init triggers
2022-03-15 15:19:29,166 INFO    MainThread:3820 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-03-15 15:19:29,166 INFO    MainThread:3820 [wandb_init.py:init():471] starting backend
2022-03-15 15:19:29,166 INFO    MainThread:3820 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-03-15 15:19:29,166 INFO    MainThread:3820 [backend.py:ensure_launched():219] starting backend process...
2022-03-15 15:19:29,291 INFO    MainThread:3820 [backend.py:ensure_launched():224] started backend process with pid: 7820
2022-03-15 15:19:29,291 INFO    MainThread:3820 [wandb_init.py:init():480] backend started and connected
2022-03-15 15:19:29,291 INFO    MainThread:3820 [wandb_init.py:init():550] updated telemetry
2022-03-15 15:19:29,385 INFO    MainThread:3820 [wandb_init.py:init():581] communicating current version
2022-03-15 15:19:31,517 INFO    MainThread:3820 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-15 15:19:31,517 INFO    MainThread:3820 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-15 15:19:31,673 INFO    MainThread:3820 [wandb_init.py:init():624] starting run threads in backend
2022-03-15 15:19:36,675 INFO    MainThread:3820 [wandb_run.py:_console_start():1827] atexit reg
2022-03-15 15:19:36,675 INFO    MainThread:3820 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-03-15 15:19:36,675 INFO    MainThread:3820 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-03-15 15:19:36,690 INFO    MainThread:3820 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-15 15:19:36,690 INFO    MainThread:3820 [wandb_init.py:init():651] run started, returning control to user process
2022-03-15 15:19:53,217 INFO    MainThread:3820 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2022-03-15 15:19:53,217 INFO    MainThread:3820 [wandb_run.py:_restore():1769] restore
2022-03-15 15:19:53,643 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1061
  total_bytes: 1061
}

2022-03-15 15:19:53,756 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1061
  total_bytes: 1061
}

2022-03-15 15:19:53,991 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1061
  total_bytes: 1061
}

2022-03-15 15:19:54,894 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1061
  total_bytes: 15162
}

2022-03-15 15:19:55,019 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1061
  total_bytes: 15162
}

2022-03-15 15:19:55,132 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1061
  total_bytes: 15162
}

2022-03-15 15:19:55,246 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15162
  total_bytes: 15162
}

2022-03-15 15:19:55,356 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15162
  total_bytes: 15162
}

2022-03-15 15:19:55,465 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15162
  total_bytes: 15162
}

2022-03-15 15:19:55,574 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15162
  total_bytes: 15162
}

2022-03-15 15:19:55,684 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15162
  total_bytes: 15162
}

2022-03-15 15:19:55,981 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15162
  total_bytes: 15162
}

2022-03-15 15:19:56,310 INFO    MainThread:3820 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15162
  total_bytes: 15162
}
local_info {
}

2022-03-15 15:19:57,728 INFO    MainThread:3820 [wandb_run.py:_append_files():2194] logging synced files
